src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -1564,7 +1565,8 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                         if ((document.NonPersistentFlags.Contain(NonPersistentDocumentFlags.FromSmuggler)) &&
                             (_missingDocumentsForRevisions != null))
                         {
-                            if (_database.DocumentsStorage.Get(context, document.Id) == null)
+                            if (_database.DocumentsStorage.Get(context, document.Id) == null && 
+                                document.ChangeVector.Contains(ChangeVectorParser.TrxnTag) == false)
```

Comment:
this is to avoid deleted revision upon restore, right?

Reply:
Yes